uniffi_bindgen/src/scaffolding/mod.rs

```diff
@@ -70,67 +68,55 @@ mod filters {
         })
     }
 
-    pub fn lower_rs(nm: &dyn fmt::Display, type_: &Type) -> Result<String, askama::Error> {
-        // By explicitly naming the type here, we help the rust compiler to type-check the user-provided
-        // implementations of the functions that we're wrapping (and also to type-check our generated code).
+    /// Get the name of the FfiConverter implementation for this type
+    ///
+    /// - For primitives / standard types this is the type itself.
+    /// - For user-defined types, this is a unique generated name.  We then generate a unit-struct
```

Comment:
nit: This "We then generate a unit-struct" feels like the first half of some larger thought, is it something that needs to be fleshed out more, or a leftover to be removed?